agent/consul/leader.go

```diff
@@ -1454,7 +1498,9 @@ func (s *Server) reapTombstones(index uint64) {
 	}
 	_, err := s.raftApply(structs.TombstoneRequestType, &req)
 	if err != nil {
-		s.logger.Printf("[ERR] consul: failed to reap tombstones up to %d: %v",
-			index, err)
+		s.logger.Error("failed to reap tombstones up to index",
+			"index", index,
+			"error", err,
+		)
 	}
```

Comment:
Skimming this I realized there's an unrelated bug here. Every `raftApply()` should have two error conditionals following it:
```
resp, err := s.raftApply(....blah....)
if err != nil {
  <do with err>
}
if respErr, ok := resp.(error); ok {
  <do with respErr>
}
```
This `raftApply` only has one.

Reply:
I think because with this we are requesting raft do something, and we don't have to do anything but hope it doesn't give us an err?